src/vconsole/vconsole-setup.c

```diff
@@ -43,14 +44,16 @@
 #include "util.h"
 #include "virt.h"
 
+#define MAX_CONSOLES 63
+
 static bool is_vconsole(int fd) {
         unsigned char data[1];
 
         data[0] = TIOCL_GETFGCONSOLE;
         return ioctl(fd, TIOCLINUX, data) >= 0;
 }
 
-static int disable_utf8(int fd) {
+static int disable_utf8(unsigned int fd) {
```

Comment:
file descriptors are plain "int"s on UNIX, you can't change that, really. If UNIX was designed today it probably would get fd_t or so instead, but since this is an "int" historically, it has to be an "int" now, too.